lib/motor/include/motor/MotorRIOT.hpp

```diff
@@ -13,6 +13,13 @@ class MotorRIOT: public MotorInterface {
     {
     }
 
+    /// @brief Init the motor and its driver
+    /// @return 0 on success, negative on error
+    int init()
+    {
+        return driver_.init();
```

Comment:
Do you check if we can safely call driver init multiple times for the same driver ?

Reply:
yes, in case of double h-bridge, the driver uses the same PWM frequency/resolution for both motors:
https://github.com/RIOT-OS/RIOT/blob/bf8c734f988c7f6686e23b3b28aee9aae9658da1/drivers/include/motor_driver.h#L168

Here we have 2 simple h-bridge, but seen as one double because we apply the same configuration.

So yes it is completly safe. Moreover if a mistake has to be done, it will be in the motor param file, so the risk is the same than doing the init outside of the class.